### PR TITLE
Update talk_basics in user manual

### PR DESCRIPTION
**Pull request by DorraJaouad**
* Added the filter conversations docs
* Added the open conversations list docs
* Updated outdated screenshots and/or their descriptions.

The screenshots have fixed widths because their sizes might appear larger than their actual dimensions when displayed.

### CODE REVIEW

File: user_manual/talk/talk_basics.rst [hunk 1]
```diff
@@ -18,18 +18,47 @@ You can create a direct, one-on-one chat by searching for the name of a user, a
 
 .. image:: images/chat-with-one-user.png
 
-If you want to create a custom group conversation, click the ``plus`` button next to the search field. You can then pick a name for the conversation and select if the conversation should be open to external users and if other users on the server can see and join the conversation.
+If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``create a new conversation``.
```
Comment (nickvergessen) [inline user_manual/talk/talk_basics.rst:21]:
```suggestion
If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``Create a new conversation``. 
```

File: user_manual/talk/images/new-room.png
Comment (Antreesy) [inline user_manual/talk/images/new-room.png:1]: Maybe it'll be better to redo this screenshot without browser extensions (visible in the NewMessage and are not the part of Talk)

File: user_manual/talk/images/join-open-conversations.png
Comment (Antreesy) [inline user_manual/talk/images/join-open-conversations.png:1]: Let's put more open chats in this list, it feels empty :hole:

File: user_manual/talk/images/devices-settings.png
Comment (Antreesy) [inline user_manual/talk/images/devices-settings.png:1]: I'd put something between a camera (plushy toy maybe :teddy_bear: ) or even use a fake camera (like OBS studio, for example)

File: user_manual/talk/images/add-participants.png
Comment (Antreesy) [inline user_manual/talk/images/add-participants.png:1]: Miss the 'Management' group on this screenshot

File: user_manual/talk/talk_basics.rst [hunk 2]
```diff
@@ -18,18 +18,47 @@ You can create a direct, one-on-one chat by searching for the name of a user, a
...
 .. image:: images/chat-with-one-user.png
 
-If you want to create a custom group conversation, click the ``plus`` button next to the search field. You can then pick a name for the conversation and select if the conversation should be open to external users and if other users on the server can see and join the conversation.
+If you want to create a custom group conversation, click the ``⋮`` button next to the filters button and then on ``create a new conversation``. 
 
-.. image:: images/creating-group-room.png
+.. image:: images/create-new-conversation.png
+    :width: 300px
+
+You can then pick a name for the conversation and select if the conversation should be open to external users and if other users on the server can see and join the conversation.
```
Comment (Antreesy) [inline user_manual/talk/talk_basics.rst:26]:
```suggestion
You can then pick a name for the conversation, put a description, and set up an avatar for it (with uploaded photo or emoji), and select if the conversation should be open to external users and if other users on the server can see and join the conversation.
```

File: user_manual/talk/talk_basics.rst [hunk 3]
```diff
@@ -104,7 +134,13 @@ You can hide your video (useful during a screen share) with the little arrow jus
...
 You can access your settings and choose a different webcam, microphone and other settings in the ``...`` menu in the top bar.
 
-.. image:: images/call-menu.png
+.. image:: images/media-settings.png
+    :width: 300px
+
+.. image:: images/devices-settings.png
+    :width: 450px
+
+You can access other settings in ``Conversation settings``.
```
Comment (Antreesy) [inline user_manual/talk/talk_basics.rst:143]:
```suggestion
You can change other settings in the ``Conversation settings`` dialog.
```

File: user_manual/talk/talk_basics.rst [hunk 4]
```diff
@@ -104,7 +134,13 @@ You can hide your video (useful during a screen share) with the little arrow jus
...
 
 .. image:: images/talk-settings.png
```
Comment (Antreesy) [inline user_manual/talk/talk_basics.rst:145]: This screenshot also could be updated, as we have a couple of new sections in this list

File: user_manual/talk/images/devices-settings.png
Comment (Antreesy) [reply user_manual/talk/images/devices-settings.png:1]: Before call | In call
--- | ---
Screenshot from 2023-08-15 16-03-42 | Screenshot from 2023-08-15 16-03-16

Comment (DorraJaouad) [reply user_manual/talk/images/devices-settings.png:1]: I used photoshop

File: user_manual/talk/talk_basics.rst [hunk 5]
```diff
@@ -104,7 +141,18 @@ You can hide your video (useful during a screen share) with the little arrow jus
 
 You can access your settings and choose a different webcam, microphone and other settings in the ``...`` menu in the top bar.
 
-.. image:: images/call-menu.png
+.. image:: images/media-settings.png
+    :width: 300px
+
+.. image:: images/device-settings-before-call.png
```
Comment (Antreesy) [inline user_manual/talk/talk_basics.rst:147]: You should put that image before `images/join-call.png`, and set a small description, i think:

```
When you're in a conversation, you can start a call any time with the ``Start call`` button. Other participants will get notified and can join the call. Before call is started, you can select and switch on/off media devices (webcam, microphone) and select a virtual background


.. image:: images/device-settings-before-call.png
:width: 300px

 If somebody else has started a call already, the button will change in a green ``Join call`` button.
.. image:: images/join-call.png
```

File: user_manual/talk/talk_basics.rst [hunk 6]
```diff
@@ -104,7 +141,18 @@ You can hide your video (useful during a screen share) with the little arrow jus
...
+    :width: 300px
+
+or change the background 
+
+.. image:: images/background-settings-in-call.png
+    :width: 300px
+
+You can change other settings in the ``Conversation settings`` dialog.
```
Comment (Antreesy) [inline user_manual/talk/talk_basics.rst:155]: This is a screenshot for `Talk settings` (bottom left corner)

File: user_manual/talk/talk_basics.rst [hunk 7]
```diff
@@ -83,13 +118,15 @@ Changing permissions of a user that joined a public conversation will also perma
...
 
-Moderators can configure the conversation. Select ``Conversation settings`` from the gear menu of the conversation on the top to access the settings.
+Moderators can configure the conversation. Select ``Conversation settings`` from the ``...`` menu of the conversation on the top to access the settings.
 
 .. image:: images/open-settings.png
+    :width: 300px
 
 Here you can configure the description, guest access, if the conversation is visible to others on the server and more.
 
 .. image:: images/chat-room-settings.png
```
Comment (Antreesy) [inline user_manual/talk/talk_basics.rst:128]: This screenshot is outdated

**Review by Antreesy — COMMENTED** [review]
Some more minor comments. Rest seems fine!

**Review by Antreesy — APPROVED** [review]
Nicely done! :tada:

**Pull request merged**